Genetic Information (DNA & RNA)
===============================

DNA Replication & Repair
------------------------

DNA Structure 
^^^^^^^^^^^^^

- **Nucleotide**: *deoxyribose, phophate, base (adenine, thymine, cytosine, guanine)*
- **Phosphodiester Bond**

  - **Junction**: *3'OH group of nucleotide 1 with 5'OH group of nucleotide 2 via phosphoryl group*
  - **Polarity**: *5'end (free PO4 group), 3'end (free OH group), sequence written from 5' to 3'*

- **Double helix**

  - **Sturcture**: *antiparallel, deoxyribose-phosphate backbone outside, base inside, twisted ladder structure, create major & minor groove*
  - **Base pairing**: *adenine with thymine, guanine with cytosine, held by hydrogen bond (2 between A-T, 3 between G-C)*
  - **Strand separation**: *separate when H-bond disrupted (denaturation), by altered pH or heat, ssDNA has higher 260nm absorption, Tm defined as temparature of 50% separation, A-T rich DNA has lower Tm*
  - **Structural form**: *B form (primary form, right-handed), A form (right-handed), Z form (left-handed)*
  - **Linear & circular form**: *eukaryotes: linear dsDNA (and circular mitochondrial DNA), prokaryotes: circular dsDNA (and plasmid)*

- **Eukaryotic DNA**

  - **Histone**: *positively charged (high lysine & arginine), bind negative charged DNA*
  - **Nucleosome**: *H2A & H2B & H3 & H4 form octameric core, dsDNA round around, H1 bind to linker DNA*
  - **Histone modification**: *acetylation & methylation & phosphorylation, affect gene expression, heritable (epigenetics)*
  - **Higher level organization**: *nucleosome packed to form nucleofilament, furture packed to form chromosome*

Prokaryotic DNA Replication 
^^^^^^^^^^^^^^^^^^^^^^^^^^^

- **General**: *semiconservative replication, each strand of separated dsDNA serve as template*
- **Complementary strand separation**: *prokaryotes: single replication origin (AT-rich consensus sequence), eukaryotes: multiple replication origin*
- **Replication fork formation**

  - **DnaA protein**: *bind and melt replication origin*
  - **DNA helicase**: *unwind double helix, require ATP, cause supercoil*
  - **Single-stranded DNA-binding protein**: *bind & protect ssDNA*

- **Solving supercoil**

  - **Type I DNA topoisomerase**: *cleave one strand, rotate around intact strand, reseal nick*
  - **Type II DNA topoisomerase**: *cleave two strand, pass another part of dsDNA, reseal break*

- **Direction of replication**: *read in 3'→5', synthesize in 5'→3'*

  - **Leading strand**: *synthesized continuously toward replication fork*
  - **Lagging strand**: *synthesized discontinuously away from replication fork, discontinuous Okazaki fragment eventually join*

- **RNA primer**: *DNA pols cannot initiate synthesis*

  - **Primase**: *synthesize short RNA complementary to DNA template, one on leading strand, multiple on lagging strand*
  - **Primosome**: *primase + prepriming complex*

- **Chain elongation**: *add nucleotide to 3'end of growing chain, complementary to template*

  - **Synthesis**: *by DNA pol III (5'→3' polymerase activity), β-subunit forming sliding clamp give rise to high processivity*
  - **Proofreading**: *by DNA pol III (3'→5' exonuclease activity), remove erroneously added nucleotide*

- **Primer excision & replacement & ligation**

  - **Excision**: *by DNA pol I (5'→3' exonuclease activity), remove RNA primer*
  - **Replacement**: *by DNA pol I (5'→3' polymerase activity), fill the gap with DNA*
  - **Ligation**: *by DNA ligase, join DNA stretches*

- **Termination**

Eukaryotic DNA Replication 
^^^^^^^^^^^^^^^^^^^^^^^^^^

- **Eukaryotic DNA polymerase**

  - **Pol α**: *contain primase*
  - **Pol β**: *gap filling in DNA repair*
  - **Pol γ**: *replicate mitochondrial DNA*
  - **Pol δ**: *elongate lagging strand*
  - **Pol ε**: *elongate leading strand, PCNA serve as sliding clamp*

- **Telomeres**

  - **Structure**: *repeated AGGGTT at end of chromosome, 3'end longer*
  - **Telomere shortening**: *no way to fill extreme 5'end of lagging strand, telomere shorten with cell division*
  - **Telomerase**: *extend 3'end of telomere, use RNA template & reverse transcriptase, DNA pol α synthesize complementary strand*

- **Others**

  - **Reverse transcriptase**: *in HIV & transposon*
  - **Nucleoside analog**: *block DNA synthesis, slow division of cancer cell or virus (ex: dideoxyinosine, arabinoside, azidothymine)*

DNA Repair 
^^^^^^^^^^

- **Mismatch repair**

  - **Error**: *incorrect base-pairing*
  - **Mismatched strand identification**: *Mut protein discriminate parental strand (methylated) & daughter strand (not methylated), daughter strand is repaired*
  - **Repair**: *mismatched nucleotides removed by endonuclease & exonuclease, gap filled by DNA pol III & DNA ligase*
  - **Disease**: *lynch syndrome (hereditary nonpolyposis colorectal cancer), increased risk of colon cancer*

- **Nucleotide excision repair**

  - **Error**: *convalent joining of adjacent pyrimidine (T dimer), caused by IV radiation*
  - **Repair**: *uvrABC recognize dimer & cleave on both side of damage, nucleotides containing dimer removed, gap filled by DNA pol I & DNA ligase*
  - **Disease**: *xeroderma pigmentosum, early & numerous skin cancer*

- **Base exicision repair**

  - **Error**: *DNA base lost or deaminated spontaneously or by chemical (nitrous acid)*
  - **Repair**: *abnormal base cleaved by DNA glycosylase, AP site cleaved by AP endonuclease, base-free residue removed by deoxyribose phosphate lyase, gap filled by DNA pol I & DNA ligase*

- **Double-strand break repair**

  - **Error**: *double-strand break caused by ionizing radiation or chemotherapeutic agent or free radical*
  - **Nonhomologous end joining**: *join ends of DNA fragment, more error prone*
  - **Homologous recombination**: *repair by homologous DNA, less error prone*

RNA Synthesis & Processing 
--------------------------

RNA Structure 
^^^^^^^^^^^^^

- **General**: *ribose instead of deoxyribose, uracil instead of thymine, single strand instead of double strand*
- **Ribosomal RNA**: *80% of RNA, component of ribosome, prokaryotes: 23S & 16S & 5S, eukaryotes: 28S & 18S & 5.8S & 5S (mitochondria: 16S & 12S)*
- **Transfer RNA**: *15% of RNA, carry amino acid for translation, at least 20 species for each amino acid, have unusual base & intrachain base pairing*
- **Messenger RNA**: *5% of RNA, carry genetic information from DNA, most heterogenous, prokaryotes: polycistronic, eukaryotes: monocistronic, have 5' methylguanosine cap & 3' poly-A tail*

Prokaryotic Gene Transcription 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- **General**: *RNA synthesized from 5' to 3', complementary to template strand & identical to coding strand (U replace T)*
- **Prokaryotic RNA polymerase**

  - **Core enzyme**: *α+α+β+β'+Ω subunits, required for transcription but cannot recognize promoter region*
  - **Holoenzyme**: *core enzyme + σ subunit, can recognize promoter region*

- **Initiation**: *RNA pol holoenzyme recognize promoter sequence*

  - **-35 sequence**: *"TTGACA" sequence 35 base above transcription strat site*
  - **-10 sequence (Pribnow box)**: *TATAAT sequence 10 base above transcription start site, site of melting*

- **Elongation**: *RNA pol add nucleotide to chain in 5'→3' direction, do not have primer or proofreading*
- **Termination**

  - **Rho independent**: *self-complementary RNA sequence form hairpin (GC-rich), facilitate separation of RNA from DNA*
  - **Rho dependent**: *rho protein with helicase activity, move along RNA to reach RNA pol and separate RNA & DNA*

- **Other**

  - **Antibiotics**: *inhibit RNA synthesis (ex: rifampin, dactinomycin)*

Eukaryotic Gene Transcription
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- **Transcription factor**: *bind to promoter region or regulatory element, assemble transcription intiation complex & determine gene to be transcribed*
- **Chromatin remodeling**: *interconversion of euchromatin (active decondensed) & heterochromatin (inactive condensed), by acetylation of histone (eliminate positive charge)*
- **RNA polymerase**

  - **RNA polymerase I**: *synthesize rRNA (28S, 18S, 5.8S)*
  - **RNA polymerase II**: *synthesize mRNA & small RNA*

    - **Core promoter element**: *binding site of general transcription factor, include TATA box & Inr & DPE*
    - **General transcription factor**: *recognize promoter element, recruit RNA pol II, initiate transcription*
    - **Regulatory element**: *binding site of specific transcription factor, include proximal (CAAT box, GC box) & distal (enhancer) regulatory element*
    - **Specific transcription factor**: *recognize regulatory element, recruit general transcription factor, regulate frequency of transcription, respond to hormone to determine gene to be expressed*

  - **RNA polymerase III**: *synthesize tRNA & rRNA (5S) & small RNA*

- **Other**

  - **RNA polymerase inhibitor**: *inhibit mRNA synthesis (ex: α-amanitin)*

Posttranscriptional Modification 
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

- **Ribosomal RNA**: *long precursor cleaved to yield pieces of rRNA (prokaryotes: 23S & 16S & 5S, eukaryotes: 28S & 18S & 5.8S)*
- **Transfer RNA**: *ending & intervening sequence removed, CCA sequence added to 3'end, base modified to form unusual base*
- **Eukaryotic messenger RNA**

  - **Addition of 5'cap**: *7-methylguanosine attatched to 5'end through 5'-5' triphosphate linkage, stabilize mRNA & permit translation*
  - **Addition of 3'-tail**: *poly-A tail attatched to 3'end after AAUAAA sequence, terminate transcription & stabilize mRNA & facilitate exit from nucleus*
  - **Splicing**

    - **General**: *remove intron (non-coding) & join exon (coding) by spliceosome*
    - **Small nuclear RNA**: *form snRNP (U1 & U2 & U4 & U5 & U6), mediate splicing*
    - **Mechanism**: *snRNP base-pair with sequence at each end of intron, intron excised by two transesterification*
    - **Alternative splicing**: *pre-mRNA spliced in alternative ways in different tissue, produce diverse protein from limited gene*
    - **Disease**: *splic site mutation lead to aberrant protein & genetic disease*